src/lib/checkbox/checkbox.scss

```diff
@@ -189,6 +190,10 @@ $_md-checkbox-indeterminate-checked-easing-function: cubic-bezier(0.14, 0, 0, 1)
 
 md-checkbox {
   cursor: pointer;
+
+  // Animation
+  transition: background $swift-ease-out-duration $swift-ease-out-timing-function,
+              md-elevation-transition-property-value();
```

Comment:
Why is `md-elevation-transition-property-value` here? This was only meant to be used with elevation